apps/web/vibes/soul/form/select/index.tsx

```diff
@@ -11,6 +11,7 @@ import { Label } from '@/vibes/soul/form/label';
 type Props = {
   id?: string;
   name: string;
+  pending?: boolean;
```

Comment:
Had to add a `pending` prop to `Select` because we spread to Radix's `Select.Root` which doesn't render a DOM element.